### ISSUE TITLE
Verbose Error Logging 

### ISSUE THREAD

**Issue by ObsidianCarnageX12** [Aleph-Bet-Marathon/alephbet#12]
It is a odd omission to not have any form of Verbose Error logging in LUA or Engine .
LUA Errors only sometimes seem to print to a log , and reading them on screen is just like reading ancient scripture.

**Comment by SolraBizna** [comment]
It would be very good to have Lua errors consistently print full stacktraces to the log file. (A full stacktrace appearing on screen is only *sometimes* useful.)

**Comment by aaronfreed** [comment]
Agreed. The stack traces in -a1-lua-shim.lua are immensely helpful, but they really should just be a standard engine feature.
